test/OutputCacheProviderUnitTests_462/packages.config

```diff
@@ -7,7 +7,7 @@
   <package id="Pipelines.Sockets.Unofficial" version="2.2.2" targetFramework="net462" />
   <package id="StackExchange.Redis" version="2.5.61" targetFramework="net462" />
   <package id="System.Buffers" version="4.5.1" targetFramework="net462" />
-  <package id="System.Diagnostics.PerformanceCounter" version="6.0.0" targetFramework="net462" />
+  <package id="System.Diagnostics.PerformanceCounter" version="6.0.1" targetFramework="net462" />
```

Comment:
Can we eliminate all packages.config files by also converting the test .csproj files to SDK style?